rs/boundary_node/rate_limits/canister/add_config.rs

```diff
@@ -0,0 +1,287 @@
+use std::collections::{BTreeMap, HashMap, HashSet};
+
+use crate::storage::StorableIncidentMetadata;
+use ic_cdk::api::time;
+use rate_limits_api::IncidentId;
+use serde_json::{Map, Value};
+use sha2::{Digest, Sha256};
+use thiserror::Error;
+
+use crate::{
+    access_control::{AccessLevel, ResolveAccessLevel},
+    state::Repository,
+    storage::{StorableConfig, StorableRuleMetadata},
+    types::{InputConfig, RuleId, Version},
+};
+
+pub const INIT_VERSION: Version = 1;
```

Comment:
What does this value mean? Keep in mind it will get reset back to 1 any time the canister is restarted.

Reply:
This is the number from which the versions start. Config/version initialization should run only once, when the canister is installed for the first time. I will handle this logic later to make sure config is never initialized twice.